spec/rubocop/ast/pair_node_spec.rb

```diff
@@ -37,20 +37,6 @@
     end
   end
 
-  describe '#colon?' do
-    context 'when using a hash rocket delimiter' do
-      let(:source) { '{ a => 1 }' }
-
-      it { expect(pair_node.colon?).to be_falsey }
-    end
-
-    context 'when using a colon delimiter' do
-      let(:source) { '{ a: 1 }' }
-
-      it { expect(pair_node.colon?).to be_truthy }
-    end
-  end
```

Comment:
These seem unrelated to the change. 🤔

Reply:
You are right, although it was needed to have the CI pass. Note that it's a separate commit. Seemed easier than create a PR just for that, wait for CI, merge it, rebase this one, push, wait for CI, merge 😆 . Let me know if it causes issues